soc/v2/nuvoton/numaker/soc.yml

```diff
@@ -0,0 +1,4 @@
+series:
```

Comment:
should this describe the family `numaker` ?

Reply:
fixed